games-emulation/pcsx2/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<herd>proxy-maintainers</herd>
+	<maintainer>
+		<email>ykonotopov@gmail.com</email>
+		<name>Yuri Konotopov</name>
+	</maintainer>
+	<use>
+		<flag name="png">Allow to dump texture as PNG (require <pkg>dev-cpp/pngpp</pkg>)</flag>
+		<flag name="wxwidgets3">Link against version 3.0 of <pkg>x11-libs/wxGTK</pkg> instead of 2.8</flag>
```

Comment:
It seems there is no way to autoselect wxGTK version in wxwidgets.eclass.
Defining WX_GTK_VER is mandatory, so for multi-dependency only [has_version variant](https://github.com/nE0sIghT/gentoo-portage-rsync-mirror/commit/07ec4139218061af6801ec7d77a918b21dc9aca6#diff-6e6824870797d59af5d017556591e434L133) is suitable as i see.